test/schema/schema_test.go

```diff
@@ -508,15 +509,28 @@ func TestSchema_integration(t *testing.T) {
 		require.Nil(t, errRm)
 	})
 
-	t.Run("Create class with explicit replication config", func(t *testing.T) {
+	t.Run("tear down weaviate", func(t *testing.T) {
+		err := testenv.TearDownLocalWeaviate()
+		if err != nil {
+			t.Fatalf("failed to tear down weaviate: %s", err)
+		}
+	})
+}
+
+/*
```

Comment:
Why is this test commented?

Reply:
Because it was constantly failing, and I'm not convinced that the go client is the right place for doing cluster testing.

Reply:
Not when combined with basic functionality tests for the client

Reply:
We can put it in its own file later

Reply:
maybe instead of commenting it, please add `t.Skip()` statement, it will be then easy to find